spec/system/admin/product_import_spec.rb

```diff
@@ -96,6 +96,8 @@
 
       wait_until { page.find("a.button.view").present? }
 
+      puts "TODO: migrate to v3"
```

Comment:
Shouldn't that be tracked in an issue ?

Reply:
Yes, I'll add checkboxes in the above issue 👍

Reply:
Done